internal/commands/login/inputs_test.go

```diff
@@ -184,6 +221,10 @@ func TestLoginInputs(t *testing.T) {
 
 			assert.Nil(t, tc.inputs.Resolve(profile, ui))
 
+			if tc.expectBrowser {
```

Comment:
I don't love wrapping test asserts within conditionals like this (I know its easy/full-proof here, but it's a slippery slope)

rather than listen to my `expectNoBrowser bool` comment above, seems like what you could actually do is have `expectedURL string` and set that to `apiKeysPage` everywhere you currently have `expectBrowser: true`

then this assert simply becomes: `assert.Equal(t, tc.expectedURL, expectedURL)`
(note: the flip of args there, the CLI likes having `expected` first then `actual` ... I have very strong feelings both ways depending on the day on whether I like that pattern or not 😬 )